packages/ra-ui-materialui/src/list/ListActions.tsx

```diff
@@ -72,12 +71,7 @@ export const ListActions = (props: ListActionsProps) => {
                     : filters && <FilterButton />}
                 {hasCreate && <CreateButton />}
                 {exporter !== false && (
-                    <ExportButton
-                        disabled={total === 0}
-                        resource={resource}
-                        sort={sort}
-                        filterValues={filterValues}
-                    />
+                    <ExportButton disabled={total === 0} resource={resource} />
```

Comment:
**question**: we no longer want `<ExportButton>` export the same data as the list?
Also and if I'm not mistaken, `sort` param seems to be not used, don't we should remove it from `ListActionsProps`?